__tests__/app.test.js

```diff
@@ -47,15 +47,17 @@ describe("app", () => {
           .get("/api/articles/1")
           .expect(200)
           .then(({ body }) => {
-            expect(body).toEqual({
-              article_id: 1,
-              title: "Living in the shadow of a great man",
-              topic: "mitch",
-              author: "butter_bridge",
-              body: "I find this existence challenging",
-              created_at: `2020-07-09T20:11:00.000Z`,
-              votes: 100,
-            });
+            expect(body.article).toEqual(
+              expect.objectContaining({
```

Comment:
This is pretty much asserting the the same thing as before.

The reason David said this was brittle is that if you change your code to add an extra bit of data to the response *spoliers* this test would immediately break.

You could instead assert something like 
expect(body.article).toBe(1)
etc.

This would test for the properties you want and also not break if more data is added!